sample_apps/run_example.sh

```diff
@@ -516,6 +621,11 @@ function rm_non_git_files() {
 # ###########################################################################
 function build_utilities() {
     run_cmd
+    if [ "${SampleAppName}" = "simple_app_under_app_service" ]; then
+        echo "${Me}: Skipping ..."
```

Comment:
Some steps are skipped while setting up the app, as they are supposed to have been "just-done" while `setup` is run for `application_service`.